.github/workflows/gradle.yml

```diff
@@ -33,4 +33,4 @@ jobs:
     - uses: actions/upload-artifact@v2
       with:
         name: Package
-        path: build/libs
\ No newline at end of file
+        path: build/libs
```

Comment:
I don't see a change here?

Reply:
I changed the distribution to zulu and added fixed versions. Temurin doesn't support some of the JDK versions your project specifies. 
:-)